src/lib/net/contracts.spec.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import '@testing-library/jest-dom'
+import { providers, Wallet } from 'ethers';
+import type { ExternalProvider } from '@ethersproject/providers';
+import { Contracts } from './contracts';
+import { deployments } from 'renpool-contracts';
+
+describe('contracts', () => {
+
+    const network = {
+        renPoolAddr: '0xbf115a5538290d234fa31e917241a58a20a847bc',
+        renTokenAddr: deployments.kovan.renTokenAddr,
```

Comment:
We should set the chain based on the currently connected chain, right? I'm referring to this line
```
deployments.kovan.renTokenAddr
```

Reply:
This was just a hardcoded value to make a PoC. That value would depend on the fork that you would start to run the test. The goal here would be to embed hardhat/ganache to start a blockchain when running those tests. But we need to figure out how to do so.